README.md

```diff
@@ -464,7 +446,7 @@ maintaining the Node.js project.
 
 ### Release Team
 
-Node.js releases are signed with one of the following GPG keys:
+Ayo.js releases are signed with one of the following GPG keys:
```

Comment:
I don’t think this is very accurate :) How about just replacing `are signed` → `were signed` for now?